### ISSUE TITLE
NTR: Treponema pallidum PCR in CSF positive (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8461]
New term request Treponema pallidum PCR in CSF positive (samant_101126232017): Cerebrospinal fluid polymerase chain reaction; syphilis; DNA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000763
- label: Positive CSF treponema pallidum nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: 
- def: Detection of Treponema pallidum (syphillis) nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8461
- synonyms:
